index.js

```diff
@@ -1,6 +1,28 @@
 /* eslint-env node */
 'use strict';
+let VersionChecker = require('ember-cli-version-checker');
+const UI = require('console-ui');
+const ui = new UI({
+  inputStream: process.stdin,
+  outputStream: process.stdout,
+  errorStream: process.stderr,
+  writeLevel: 'INFO',
+  ci: true
+});
+
 
 module.exports = {
-  name: 'ember-qunit-nested-module-blueprints-polyfill'
+  name: 'ember-qunit-nested-module-blueprints-polyfill',
+  isEnabled() {
+    let checker = new VersionChecker(this);
+    let dep = checker.forEmber();
+    let deprecated = dep.gte('3.0.0');
+    if (deprecated) {
+      ui.writeError('ember-qunit-nested-module-blueprints-polyfill is no longer needed in Ember 3.0. ');
```

Comment:
Inside the addon you can do `this.ui`

Reply:
awesome... couldn't find it in https://ember-cli.com/api/classes/Addon.html but I found it in the source code now that I was looking for it.

Is there a golden path for testing this by instantiating a new project with an addon like `ember-cli-addon-tests` or `ember-cli-blueprint-test-helpers`?